### ISSUE TITLE
Memory

### ISSUE THREAD

**Issue by mnater** [mnater/Hyphenopoly#69]
The memory allocated by Hyphenopoly_Loader and Hyphenopoly currently doesn't get freed.

![img1](https://user-images.githubusercontent.com/280635/55295622-91456e80-540f-11e9-98f0-7a5c8a9bff85.png)

That's ok if at some point later Hyphenopoly is used again on a specific site (e.g. if dynamically loaded text need to be hyphenated), if not the memory should get freed.

We could do

````javascript
H.events.define(
            "hyphenopolyEnd",
            function def() {
                // […]
                window.Hyphenopoly = null;
            },
            false
        );
````

(preferably behind a config tag `keepAlive`) but the elements treaten by `function registerOnCopy` keep a reference to window, thus the memory will not be garbage collected (memory leak).

![img2](https://user-images.githubusercontent.com/280635/55295758-2c8b1380-5411-11e9-918d-8a82b0ac65b3.png)

**Comment by mnater** [comment]
fixed in 8a785bafc2558f22c9c63c872cfc70fe3f4b7bf8

**Comment by mnater** [comment]
Uh!
https://github.com/mnater/Hyphenopoly/commit/8a785bafc2558f22c9c63c872cfc70fe3f4b7bf8 didn't actually implement freeing memory…
